serverlessworkflow/modules/ROOT/pages/cloud/operator/enabling-jobs-service.adoc

```diff
@@ -0,0 +1,170 @@
+= Managing Jobs Service with the Operator
+:compat-mode!:
+// Metadata:
+:description: Configure Jobs Service using the `SonataFlowPlatform` CR.
+:keywords: sonataflow, serverless, operator, kubernetes, jobs service
+
+
+This document describes how to configure the Jobs Service instance usimg the SonataFlowPlarform CR.
+
+== Automate the Jobs Service instance management with the `SonataFlow` Operator
+
+While it is possible to deploy the Jobs Service manually, the Operator also provides a convenient way to combine it with the
+namespace configuration via the `SonataFlowPlatform` CR. With this approach, the Operator will take care of configuring the
+Jobs Service instance and ensure it is in sync with the specification in the CR. When the operator manages the Jobs Service lifecycle,
+it will inject properties in `SonataFlow` workflows at creation time to enable the workflows to communicate with the Jobs Service
+It is possible to deploy the Jobs Service manually, but the Operator also provides a convenient way to combine it with the namespace configuration via the `SonataFlowPlatform` CR. This approach allows the Operator to configure the Jobs Service and make sure it is in line with the Specification in CR. When the Operator manages the jobs service lifecycle, it injects properties at creation time in the `SonataFlow` workflows to communicate with the Jobs Service during execution, eliminating the need to include these properties in the workflow CR instance of the `SonataFlow` workflow.
```

Comment:
@kaldesai not sure if you saw this, can you please take a look?

Reply:
@ricardozanini, How about now?
I tried to minimize the content without changing the meaning of the paragraph. Let me know WDYT :) Otherwise, we can change it again. 
 
`It is possible to deploy the Jobs Service manually, leveraging the Operator offers a more seamless integration by combining it with namespace configuration through the SonataFlowPlatform Custom Resource (CR). When the Operator oversees the lifecycle of the jobs service, it automatically injects necessary properties during creation into the SonataFlow workflows. This integration eliminates the need for including these properties within the SonataFlow workflow CR instance, simplifying workflow management.`

Reply:
Done